app/helpers/course/assessment/submission/submissions_guided_helper.rb

```diff
@@ -21,8 +32,7 @@ def guided_nav_class(step)
     ''
   end
 
-  # The question on current step.
-  def guided_current_question
-    @questions_to_attempt.first
+  def guided_current_answer
+    @answers.last
```

Comment:
@lowjoel Actually no need ? because `load_or_create_answers` for guided submission will only load answers for `current_question`